src/lnxLib/Utils/Input.lua

```diff
@@ -70,16 +73,16 @@ local KeyValues = {
 
 -- Fill the tables
 local function D(x) return x, x end
-for i = 1, 10 do KeyNames[i], KeyValues[i] = D(tostring(i - 1)) end -- 0 - 9
-for i = 11, 36 do KeyNames[i], KeyValues[i] = D(string.char(i + 54)) end -- A - Z
-for i = 37, 46 do KeyNames[i], KeyValues[i] = "KP_" .. (i - 37), tostring(i - 37) end -- KP_0 - KP_9
-for i = 92, 103 do KeyNames[i] = "F" .. (i - 91) end
+for i = KEY_0, KEY_9 do KeyNames[i], KeyValues[i] = D(tostring(i - KEY_0)) end
+for i = KEY_A, KEY_Z do KeyNames[i], KeyValues[i] = D(string.char(i - KEY_A + 65)) end
+for i = KEY_PAD_0, KEY_PAD_9 do KeyNames[i], KeyValues[i] = "KP_" .. (i - KEY_PAD_0), tostring(i - KEY_PAD_0) end
+for i = KEY_F1, KEY_F12 do KeyNames[i] = "F" .. (i - KEY_F1 + 1) end
 
 -- Returns the name of a keycode
 ---@param key integer
----@return string?
+---@return string
 function Input.GetKeyName(key)
-    return KeyNames[key]
+    return KeyNames[key] or "UNKNOWN"
```

Comment:
The user should be choosing how to handle unknown keys. So I prefer returning `nil` instead.